model/AI/Vocabularies/EnergyConsumptionDescriptionUnitType

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumptionDescriptionUnitType
+
+## Summary
+
+Specifies the unit of energy consumption
+
+## Description
+
+List different acceptable units of energy consumption. If the unit in which the energy consumption has been recorded is not available here, please select other.
+
+## Metadata
+
+- name: EnergyConsumptionDescriptionUnitType
+
+## Entries
+
+- kilowatt-hour: KiloWatt per hour
```

Comment:
Thank you @rgopikrishnan91 . Two things here:

1) The entry name `kilowatt-hour` may not follow the naming convention (as it contains `-` dash). We may need to change it to either `kWh` or `kilowatthour` (the latter is following the spelling in the glossary of U.S. Energy Information Administration). Also when watt is written in full, w is lowercase (except when it starts the sentence), but its symbol is an uppercase W.

2) The entry definition may need to be something along the line of "Kilowatt for one hour" and not "Kilowatt per hour". See my comment at [#discussion_r1552407399](https://github.com/spdx/spdx-3-model/pull/697#discussion_r1552407399). 

If we prefer conciseness, we can have something like `kWh: Kilowatt-hour`.

Refs:
- [Glossary - U.S. Energy Information Administration](https://www.eia.gov/tools/glossary/index.php?id=K)
- [kilowatt-hour - Wikipedia](https://en.wikipedia.org/wiki/Kilowatt-hour)

Reply:
would it be `kilowattHour`?  uppercase H - otherwise agree with the suggestion

Reply:
We can go with `kilowattHour: Kilowatt-hour.` as well.